mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -410,6 +410,16 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
         // Get the plan from the chat history
         const plan = chatHistoryManager.getDisplayOptimizedHistory().filter(message => message.type === 'openai message:agent:planning')
 
+        // Move to the last cell of the notebook
+        // We don't want to overwrite any code in the notebook
+        const notebook = notebookTracker.currentWidget?.content;
+        if (notebook) {
+            notebook.activeCellIndex = notebook.widgets.length - 1;
+        }
```

Comment:
This is a good simplification for now. 

In the near term we might want to let the agent just add new code cell in the place the user prompts the agent from. 

In the longer-term, I think we'll want to be smart about where the code we are generating goes. Agent will become not just a tool for creating new code, but also for editing existing code probably.